lib/ts/controllers/s-modal.ts

```diff
@@ -153,58 +153,69 @@ namespace Stacks {
         }
 
         /**
-         * Binds tab presses on tabbable items such that tabbing only works within the modal
+         * Gets all elements that could be tabbed to in 
          */
-        private handleFocusableElements() {
-            // get all tabbable items
-            var allTabbables = Array.from(this.modalTarget.querySelectorAll("[href], input, select, textarea, button, [tabindex]"))
+        private getAllTabbables() {
+            return <HTMLElement[]>Array.from(this.modalTarget.querySelectorAll("[href], input, select, textarea, button, [tabindex]"))
```

Comment:
optional: Didn't test this locally, but `querySelectorAll` has a generic version to set the return type:

```suggestion
            return Array.from(this.modalTarget.querySelectorAll<HTMLElement>("[href], input, select, textarea, button, [tabindex]"))
```